app/controllers/activities_controller.rb

```diff
@@ -32,6 +32,11 @@ def teams
   end
   helper_method :teams
 
+  def last_season_teams
+    Team.select {|t| t.season_id == Season.current.id + 1 && (t.kind == 'sponsored' || t.kind == 'voluntary' || t.kind == 'full_time' || t.kind == 'part_time')}
+  end
+  helper_method :last_season_teams
+
```

Comment:
Good point @F3PiX 👍 

I just added a migration align the past seasons' teams with the current datamodel in #946 , but we're still discussing whether or not it's a good idea to do it. For the sake of keeping the complexity in a sane scope, I'd personally prefer to migrate the existing data and then simply use `accepted` again.

Otherwise yes, looking for activities would be a valid option (plus I'd probably also prefer it to listing all ancient "kinds").